tutorials/metrics/metrics_file_regex_with_linenum.yml

```diff
@@ -0,0 +1,27 @@
+buildspecs:
+  metric_file_regex:
+    executor: generic.local.bash
+    type: script
+    description: capture result metric from file path
+    run: echo -e "HPCG result is INVALID with a GFLOP/s rating of=28.1215\n
+                  HPCG result is VALID with a GFLOP/s rating of=68.9888" > hpcg.txt
+    tags: tutorials
+    metrics:
+      second_line:
+        type: float
+        file_regex:
+          exp: '(\d+\.\d+)$'
+          linenum: 1
+          file: hpcg.txt
+      without_linenum:
+        type: float
+        file_regex:
+          exp: '(\d+\.\d+)$'
+          file: hpcg.txt
+    status:
+      assert_eq:
+        comparisons:
+        - name: second_line
+          ref: 68.9888
+        - name: without_linenum
+          ref: 28.1215
```

Comment:
if i understand both of these checks will pass and the point of this is to show that regex with and without line-number using same regular expression can impact since it finds the first occurence of pattern when using the expression `(\d+\.\d+)`

Reply:
right, anything you think I need to fix for this comment?